pkg/httpserver/httpserver.go

```diff
@@ -5,13 +5,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/covid19cz/erouska-backend/pkg/logging"
 	"net"
 	"net/http"
 	"strconv"
 	"time"
 
 	"github.com/covid19cz/erouska-backend/internal/hello"
-	"github.com/covid19cz/erouska-backend/internal/logging"
```

Comment:
@jendakol proc se tohle presunulo z internal do pkg? Mi moc nedava smysl mit nas custom logging exposnutej do pkg, kde to pak muze vybizet ostatni aby si ho naimportovali do svych projektu.